packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -43,8 +81,191 @@ const ReleasesPage = () => {
+interface ReleasesGridProps {
+  sectionTitle: 'pending' | 'done';
+  releases?: GetReleases.Response['data'];
+  isError?: boolean;
+}
+
+const LinkCard = styled(Link)`
+  display: block;
+`;
+
+const ReleasesGrid = ({ sectionTitle, releases = [], isError = false }: ReleasesGridProps) => {
+  const { formatMessage } = useIntl();
+
+  if (isError) {
+    return (
+      <Flex paddingTop={8}>
+        <AnErrorOccurred />
+      </Flex>
+    );
+  }
+
+  if (releases?.length === 0) {
+    return (
+      <EmptyStateLayout
+        content={formatMessage(
+          {
+            id: 'content-releases.page.Releases.tab.emptyEntries',
+            defaultMessage: 'No releases',
+          },
+          {
+            target: sectionTitle,
+          }
+        )}
+        icon={<EmptyDocuments width="10rem" />}
+      />
+    );
+  }
+
+  return (
+    <Grid gap={4}>
+      {releases.map(({ id, name }) => (
+        <GridItem col={3} s={6} xs={12} key={id}>
+          <LinkCard href={`content-releases/${id}`} isExternal={false}>
+            <Flex
+              direction="column"
+              justifyContent="space-between"
+              padding={4}
+              hasRadius
+              background="neutral0"
+              shadow="tableShadow"
+              height="100%"
+              width="100%"
+              alignItems="start"
+            >
+              <Typography as="h3" variant="delta" fontWeight="bold">
+                {name}
+              </Typography>
+            </Flex>
+          </LinkCard>
+        </GridItem>
+      ))}
+    </Grid>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleasesPage
+ * -----------------------------------------------------------------------------------------------*/
+const ReleasesPage = () => {
+  const [addReleaseDialogIsShown, setAddReleaseDialogIsShown] = React.useState(false);
+  const { formatMessage } = useIntl();
+  const [{ query }, setQuery] = useQueryParams<GetAllReleasesQueryParams>();
+  const response = useGetReleasesQuery(query);
+
+  const { isLoading, isSuccess, isError } = response;
+
+  const toggleAddReleaseDialog = () => {
+    setAddReleaseDialogIsShown((prev) => !prev);
+  };
+
+  if (isLoading) {
+    return (
+      <ReleasesLayout onClickAddRelease={toggleAddReleaseDialog} isLoading>
+        <ContentLayout>
+          <LoadingIndicatorPage />
+        </ContentLayout>
+      </ReleasesLayout>
+    );
+  }
+
+  const totalEntries = (isSuccess && response.currentData?.meta?.pagination?.total) || 0;
+
+  const handleTabChange = (index: number) => {
+    setQuery({
+      ...query,
+      page: 1,
+      pageSize: response?.currentData?.meta?.pagination?.pageSize || 16,
+      filters: {
+        $and: [
+          {
+            releasedAt: {
+              $notNull: index === 0 ? false : true,
+            },
+          },
+        ],
+      },
```

Comment:
This is not bad, but is not really necessary to pass a complex query with $and conditions for this case, because we are only filtering by one field, so I think is fine if you do:
```
filters: {
  releasedAt: {
    $notNull: conditional
  }
}
```